### ISSUE TITLE
Compatibility with storing data on Google Drive

### ISSUE THREAD

**Issue by falexwolf** [laminlabs/lamindb#263]
@Koncopd, let's discuss how much value this would add for PhD students and academia!

**Comment by Koncopd** [comment]
Interesting, there is even a fsspec solution for that https://github.com/fsspec/gdrivefs. But maybe not stable enough.

**Comment by Koncopd** [comment]
Yes, but this is not implemented by fsspec, it is a third party drivers, like most of drivers.
